lms/templates/user_dropdown.html

```diff
@@ -20,7 +20,7 @@
             <img class="menu-image" src="${profile_image_url}" alt="">
             ${username}
         </a>
-        <div role="group" aria-label="User menu" class="user-menu">
+        <div role="group" aria-label="User menu">
```

Comment:
I'm not sure why these classes are being removed. My recollection is that they were added to fix the bug with the drop down rendering on pattern library pages. In fact, I'm generally a bit confused by the four different commits on this PR that have the same names as the previous commits from the PR that introduced the problem (https://github.com/edx/edx-platform/pull/13301). Did the previous PR get reverted and then this is reapplying the changes?